x-pack/legacy/plugins/apm/public/index.tsx

```diff
@@ -9,7 +9,6 @@ import ReactDOM from 'react-dom';
 import { npStart } from 'ui/new_platform';
 import 'react-vis/dist/style.css';
 import 'ui/autoload/all';
-import 'ui/autoload/styles';
```

Comment:
Oh man. This feels nice! What about `import 'ui/autoload/all';` and `import { uiModules } from 'ui/modules';` that are currently only loaded for their side effects?

Reply:
We will not bring those into the New Platform since they will be obsolete due to no more global shared Angular instance will be in the system.